src/main/java/mate/academy/bookstore/service/user/impl/UserServiceImpl.java

```diff
@@ -23,7 +25,7 @@ public UserResponseDto register(UserRegistrationRequest request) throws Registra
         }
         User user = userMapper.toModel(request);
         user.setEmail(request.getEmail());
-        user.setPassword(request.getPassword());
+        user.setPassword(passwordEncoder.encode(request.getPassword()));
         return userMapper.toDto(userRepository.save(user));
```

Comment:
I cannot see you set any role to user during registration. Thus I don't think new user may access required endpoints